resources/zib/zib-TNMTumorClassification.DistantMetastasisLocalization.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-TNMTumorClassification.DistantMetastasisLocalization" />
```

Comment:
This profile is not referred to in any other profile.

Reply:
Ideally this concept would be mapped on `Observation.bodySite` in the M_DistantMetastasis profile, but because of the cardinality 0..* within the zib (and the cardinality of 0..1 of `Observation.bodySite`) this wouldn't work. Have you considered:

- adding an extension on `.bodysite` in the M_DistantMetastasis profile to support additional anatomical locations?
- mapping this concept on `.component` in the M_DistantMetastasis profile instead?

If however, we conclude that a separate profile is more suitable, it should be linked from the M_DistantMetastasis profile (for instance via `.hasMember`). Moreover, I think it would be better in that case to map this concept on `.valueCodeableConcept` instead of `.bodySite` within the DistantMetastasisLocalization profile. By doing so the `.value` actually corresponds to the `.code` of the Observation.